core/chain-configs/src/genesis_config.rs

```diff
@@ -25,9 +25,13 @@ use near_runtime_configs::RuntimeConfig;
 
 const MAX_GAS_PRICE: Balance = 10_000_000_000_000_000_000_000;
 
+#[cfg(feature = "protocol_feature_evm")]
+/// See https://github.com/ethereum-lists/chains/blob/master/_data/chains/1313161556.json
+pub const BETANET_EVM_CHAIN_ID: u128 = 1313161556;
```

Comment:
This isn't registered upstream yet, but I will take care of that.

Reply:
Yes, I just added it here, as I assumed somebody would add it, also the weird thing is the other two links seem non-existent currently as well.

Reply:
Submitted upstream at: https://github.com/ethereum-lists/chains/pull/145

Reply:
Merged upstream, and published at https://chainid.network

<img width="743" alt="Screen Shot 2021-01-14 at 00 33 47" src="https://user-images.githubusercontent.com/4963/104519016-638be280-5601-11eb-850b-eb4a6626398e.png">

Reply:
> also the weird thing is the other two links seem non-existent currently as well.

Upstream renamed the files: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-1313161556.json

I'll adjust the URLs in our comments the next time we touch this code.